include/circt/Dialect/Seq/SeqOpInterfaces.td

```diff
@@ -24,6 +24,15 @@ def Clocked : OpInterface<"Clocked"> {
   ];
 }
 
+def ClockTransformation : OpInterface<"ClockTransformation"> {
```

Comment:
Do we need a trait or just a function?  Are other dialects getting in on the action?  Are there times we just care about `ClockTransformation` ops but not what they do?